packages/js-util-concurrency/project.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "../../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "packages/js-util-concurrency/src",
+  "projectType": "library",
```

Comment:
shouldn't this be a `publishable` lib instead of just a `buildable` lib?

Reply:
You mean there should be a `publish` command?

Reply:
when creating a new lib with Nx you can configure it to be `publishable` and then Nx will generate some additional files and a publish command I think.

Reply:
There was no `publishable` flag for `workspace:library`. 🤷‍♂️ `js:library` has this, however.

Fixed.